pkg/codegen/python/gen.go

```diff
@@ -867,8 +865,19 @@ func (mod *modContext) importResourceType(r *schema.ResourceType) string {
 		name = "provider"
 	}
 
-	components := strings.Split(modName, "/")
-	return fmt.Sprintf("from %s%s import %s", importPath, name, components[0])
+	modName := mod.tokenToModule(tok)
+	importPath := mod.getRelImportFromRoot(modName)
+	if importPath == "." {
```

Comment:
I think we can skip adding `name` to the import path and import from the package since we generated imports in the package `__init__.py`.

So 

```
from ..childa import Member1
from . import Member1
```

instead of
```
from ..childa.member1 import Member1
from . import Member1
```